### ISSUE TITLE
TriStateCheckbox: Missing id prop

### ISSUE THREAD

**Issue by itismohammadreza** [primefaces/primereact#8018]
### Describe the bug

The TriStateCheckbox component does not expose or allow passing an id/inputId prop, making it impossible to link a to the checkbox for accessibility (using htmlFor). As a result, clicking on the label does not focus or toggle the checkbox.

### Reproducer

https://stackblitz.com/edit/zxsnnqtu?file=src%2FApp.jsx

### System Information

```Shell
Browsers:
    Edge: Chromium (131.0.2903.63)
    Internet Explorer: 11.0.26100.1882
  npmPackages:
    primereact: ^10.8.5 => 10.8.5
    react: ^18.2.0 => 18.2.0
    tailwindcss: ^3.4.4 => 3.4.4
```
